cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -866,32 +873,39 @@ protected int resumeSchedule(String namespace, String appName, String schedule)
     return response.getStatusLine().getStatusCode();
   }
 
-  protected List<ScheduleSpecification> getSchedules(String namespace, String appName,
-                                                     String workflowName) throws Exception {
+  protected List<ScheduleDetail> getSchedules(String namespace, String appName,
+                                              String workflowName) throws Exception {
     String schedulesUrl = String.format("apps/%s/workflows/%s/schedules", appName, workflowName);
     String versionedUrl = getVersionedAPIPath(schedulesUrl, Constants.Gateway.API_VERSION_3_TOKEN, namespace);
     HttpResponse response = doGet(versionedUrl);
     String json = EntityUtils.toString(response.getEntity());
-    return GSON.fromJson(json, new TypeToken<List<ScheduleSpecification>>() { }.getType());
+    return GSON.fromJson(json, new TypeToken<List<ScheduleDetail>>() { }.getType());
```

Comment:
Can we use `SCHEDULE_DETAILS_TYPE`, to avoid this extra object creation each time?